docker/user-handler.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
I agree there should be some wrapper scripts to interact with the containers, however would it be possible to generalise it to a single script? I'm thinking something like

```
#!/bin/bash
docker-compose run <scout_cli_container> "$@"
```

Once again, maybe it would be enough with one container managing all the CLI stuff, and its entrypoint could be something like 'scout -h $MONGOHOST -p $MONGOPORT'

Reply:
I can think a bit about how to do this in a good way without making it too complicated. But I agree, it would be nice to make the user interaction easier.